### PR TITLE
Hero

### LINKED ISSUE #25: Create Homepage Hero component
Create the full height hero. Add the color gradient background and the "ACM Merch" title and button to move to products. You are not making the header component that renders nav links page.

DoD:
- Add color gradient to background
- Add website title with font size (text-8xl)
- Add shop now button that scrolls down to products section

### PR DESCRIPTION
**Pull request by zijunw04**
added 
- Hero section of the website
- changed the contributor page to when user clicks contact
- added custom color for the theme of the website
- added text shadow to the theme
- changed the logo of sun/moon so they are white instead of black.

### CODE REVIEW

File: app/components/hero.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+export default function HeroScre() {
```
Comment (Ajknight121) [inline app/components/hero.jsx:3]: Include the full of the function

File: app/components/header.jsx [hunk 2]
```diff
@@ -8,11 +8,11 @@ import moon from "../images/moon.png"
 
 export default function Header({ isDarkMode, setIsDarkMode }) {
     return (
-        <div className="flex items-center justify-between">
+        <div className="flex items-center justify-between w-screen absolute">
             <div className="font-bold text-lg mt-4">
                 <Link className="text-2xl ml-6 mr-10 text-white hover:underline" href="/">Shop</Link>
                 <Link className="text-2xl mr-10 text-white hover:underline" href="https://acm.uic.edu/about">About</Link>
-                <Link className="text-2xl text-white hover:underline w" href='https://acm.uic.edu/about'>Contact</Link>
+                <Link className="text-2xl text-white hover:underline w" href='/Contributors'>Contact</Link>
```
Comment (Ajknight121) [inline app/components/header.jsx:15]: Keep the contact link to https://acm.uic.edu/contact. The link the website contributors will be added to the footer.

**Review by Ajknight121 — CHANGES_REQUESTED** [review]
Looks awesome but the Dark mode red background theme should transition back to the ACM-Dark color.

File: app/components/hero.jsx (hunk 1 above)
Comment (zijunw04) [reply app/components/hero.jsx:3]: this was fixed on the footers branch

File: app/components/header.jsx (hunk 2 above)
Comment (zijunw04) [reply app/components/header.jsx:15]: i made a commit to the footer branch which changed it back to acm.uic.edu/contact

**Pull request merged**